src/main/java/com/aws/greengrass/clientdevices/auth/CertificateManager.java

```diff
@@ -183,6 +187,7 @@ public void subscribeToCertificateUpdates(GetCertificateRequest getCertificateRe
                 };
                 subscribeToClientCertificateUpdatesNoCSR(getCertificateRequest, keyPair.getPublic(), consumer);
             }
+            metrics.incrementSubscribeSuccess();
```

Comment:
remove the word `increment`. I think just saying "subscribeSuccess" reasonably exemplifies the goal